### PR TITLE
Add Sequence Diagram for Stats, Allocate, and ViewQuota Commands

### PR DESCRIPTION
**Pull request by ThisisXXZ**
main changes in include:
* Sequence Diagrams for ``stats``, ``viewQuota`` commands
* Description for ``stats``, ``viewQuota``, and ``allocator`` commands

### CODE REVIEW

File: docs/UML_Diagrams/ViewQuotaSequence.drawio.svg
Comment (isaacsaw25) [inline docs/UML_Diagrams/ViewQuotaSequence.drawio.svg:1]: Do close the gap in the activation bar for the `:viewQuoteCommand` 😅

File: docs/UML_Diagrams/StatSequence.drawio.svg
Comment (isaacsaw25) [inline docs/UML_Diagrams/StatSequence.drawio.svg:1]: Do close the gap in the activation bar for the `:StatCommand` 😅

**Review by isaacsaw25 — CHANGES_REQUESTED** [review]
As per comments, minor changes to be made 😈 😮

File: docs/UML_Diagrams/ViewQuotaSequence.drawio.svg
Comment (paulktham) [inline docs/UML_Diagrams/ViewQuotaSequence.drawio.svg:1]: In between spotsLeft:int returning and calling getFullName() there should be a gap in the university activation block

**Review by isaacsaw25 — APPROVED** [review]
Looks good for now 👌 Might have to look into resolving the transparent background, as it may cause compatibility issues 😓

**Review by paulktham — APPROVED** [review]
nice changes!

**Comment by ThisisXXZ** [comment]
> Looks good for now 👌 Might have to look into resolving the transparent background, as it may cause compatibility issues 😓

The background is correctly displayed in both the markdown preview and the DG website, so I guess it is OK by now?

**Pull request merged**